third_party/2and3/mypy_extensions.pyi

```diff
@@ -1,8 +1,21 @@
-from typing import Dict, Type, TypeVar, Union
+from typing import Dict, Type, TypeVar, Optional, Union
 
 _T = TypeVar('_T')
 
 
+def Arg(typ: _T = ..., name: Optional[str] = ...) -> _T: ...
+
+def DefaultArg(typ: _T = ..., name: Optional[str] = ...) -> _T: ...
+
```

Comment:
Another style nit: remove empty lines between functions (this is a typeshed convention).